#driver motor values in RPM
float32 motorFLRPM
float32 motorFRRPM
float32 motorRLRPM
float32 motorRRRPM